analysed-packages/openvpn/version-2.5.6/openvpn-2.5.6-SPDX2TV.spdx

```diff
@@ -0,0 +1,7244 @@
+Copyright (C) 2017-2022 OpenVPN Inc <sales@openvpn.net> </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/contrib/pull-resolv-conf/client.up
+SPDXID: SPDXRef-item546991
+FileChecksum: SHA1: a536b86aa8ba0fe1cdd39ec8276292c150c1527c
+FileChecksum: SHA256: b9250357ecab441f941a57f5192bb0b5b57cab19c317dc1a49666c2c24a018d8
+FileChecksum: MD5: 18aa36295bc2d9bee0866e95bbc70c03
+LicenseConcluded: LicenseRef-GPL-2.0-only
+LicenseComments: <text> The information in the file is:
+
+Licensed under the GPL version 2
+
+Thus, we concluded GPL-2.0-only. </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-only
+FileCopyrightText: <text> Copyright (c) 2005-2018 OpenVPN Inc </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/contrib/pull-resolv-conf/client.down
+SPDXID: SPDXRef-item546992
+FileChecksum: SHA1: 5ec018d90fbecce6d9df7bc9ae6253f178e0d0ec
+FileChecksum: SHA256: b83d6c8b0c38bad3cf58ca4180f6cee4b2a3f29091d5a5229654fa8aa349281d
+FileChecksum: MD5: c0c55cbaaee5ac875ff52c09fd75f506
+LicenseConcluded: LicenseRef-GPL-2.0-only
+LicenseComments: <text> The information in the file is:
+
+Licensed under the GPL version 2
+
+Thus, we concluded GPL-2.0-only. </text>
+LicenseInfoInFile: LicenseRef-GPL-2.0-only
+FileCopyrightText: <text> Copyright (c) 2005-2018 OpenVPN Inc </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/build/Makefile.in
+SPDXID: SPDXRef-item546963
+FileChecksum: SHA1: 68c5c0cafce86e25b01350af451f083507e2232b
+FileChecksum: SHA256: 6d0bfa7f47c3d54c6c379c1d6a2e1da8e871ed8ad3d857d3da3c110edc598c20
+FileChecksum: MD5: bdf5eaf79e5eb48ac6b1f3d0926a054e
+LicenseConcluded: LicenseRef-FSFULLRWD
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-FSFULLRWD
+FileCopyrightText: <text> Copyright (C) 1994-2020 Free Software Foundation, Inc.
+Copyright (C) 2002-2022 OpenVPN Inc <sales@openvpn.net> </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/build/msvc/Makefile.in
+SPDXID: SPDXRef-item546954
+FileChecksum: SHA1: 8e1ddaa709540c04d7f730a61101f9f145f35640
+FileChecksum: SHA256: a00da2d0e4935925e45eded043bfeb375d559bf88833cbc22b8b232c867ad771
+FileChecksum: MD5: d14006690f68777c4cafaf150915f3f6
+LicenseConcluded: LicenseRef-FSFULLRWD
+LicenseInfoInFile: LicenseRef-FSFULLR
+LicenseInfoInFile: LicenseRef-FSFULLRWD
+FileCopyrightText: <text> Copyright (C) 1994-2020 Free Software Foundation, Inc.
+Copyright (C) 2002-2022 OpenVPN Inc <sales@openvpn.net>
+Copyright (C) 2006-2012 Alon Bar-Lev <alon.barlev@gmail.com> </text>
+
+##File
+
+FileName: openvpn-2.5.6.tar.gz/openvpn-2.5.6.tar/openvpn-2.5.6/aclocal.m4
+SPDXID: SPDXRef-item546609
+FileChecksum: SHA1: e1a5d77622a34c064f802dd7fe69282526fee53c
+FileChecksum: SHA256: f9f51ca6f6e28f6b10a993d2d2035b80c0dde827b7e08e135102650ee5422dcb
+FileChecksum: MD5: d7d5b4bb0c495a981b33d1a98ef71118
+LicenseConcluded: LicenseRef-FSFULLR AND LicenseRef-FSFULLRWD
```

Comment:
The conclusion of LicenseRef-FSFULLRWD is nearly correct, but the text is:
 This file is free software; the Free Software Foundation
 gives unlimited permission to copy and/or distribute it,
 with or without modifications, as long as this notice is preserved.

 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY, to the extent permitted by law; without
 even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 PARTICULAR PURPOSE.

and not This Makefile.in is free software; the ....
Can you please correct it.